backend/chat-backend/src/main/java/pl/wsb/chat/infrastructure/local/MessageRepositoryImpl.java

```diff
@@ -0,0 +1,9 @@
+package pl.wsb.chat.infrastructure.local;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+import pl.wsb.chat.domain.message.Message;
+import pl.wsb.chat.domain.message.MessageRepository;
+
+public interface MessageRepositoryImpl extends MessageRepository, MongoRepository<Message, String> {
```

Comment:
I think 'Impl' doesn't look good in interface. Maybe change this to MessageRepositoryLocal?